### DISCUSSION TITLE
How to run a command? How to change job attributes and library list

### DISCUSSION THREAD

**Discussion by lgiammattei** [codefori/vscode-ibmi#1231] [General]
Hi everybody, i'm trying to figure out if i can leave from RDi to move to Code for IBM i.
Mostly i work with rpgle and clle, i also have to manage dds file souce members for display files, printer files and so on.
In my Power box we have different environments and since the days of PDM we developed a smal command on our own the setup the right evinroment for the developers, this command named SETJBDLIBL (set job description attributes and library list) has only two parameters the name of the job description and the name of the library where it stays.
How can i run such command from Code for IBM i?

**Comment by martintosney** [comment]
You're in luck. Liam has just recently introduced "Command Profiles" for this exact requirement.

https://github.com/halcyon-tech/vscode-ibmi/pull/1123

I believe this is due to be in the next release (1.8.0).

**Comment by chrjorgensen** [comment]
@martintosney You just beat me to it by 30 seconds...!! 😆 

@lgiammattei The feature in #1123 has been merged and is now in the master branch - so if you want to, you can clone the repository and run the extension from here to see it in real life, before the next release.
